src/test/java/hudson/plugins/sshslaves/SSHLauncherCasCRoundTripTest.java

```diff
@@ -0,0 +1,37 @@
+package hudson.plugins.sshslaves;
+
+import hudson.model.Node;
+import hudson.slaves.SlaveComputer;
+import io.jenkins.plugins.casc.misc.ExportImportRoundTripAbstractTest;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class SSHLauncherCasCRoundTripTest extends ExportImportRoundTripAbstractTest {
+    @Override
+    protected void assertConfiguredAsExpected(RestartableJenkinsRule restartableJenkinsRule, String s) {
+        final Node node = r.j.jenkins.getNode("this-ssh-agent");
+        assertNotNull(node);
+
+        SlaveComputer computer = (SlaveComputer) node.toComputer();
+        assertNotNull(computer);
+
+        SSHLauncher launcher = (SSHLauncher) computer.getLauncher();
+        assertNotNull(launcher);
+
+        assertEquals("ssh-host", launcher.getHost());
+        assertEquals(2222, launcher.getPort());
+        assertEquals("-DuberImportantParam=uberImportantValue", launcher.getJvmOptions());
+    }
+
+    @Override
+    protected String stringInLogExpected() {
+        return "hudson.slaves.DumbSlave.name = this-ssh-agent";
+    }
+
+    @Override
+    protected String configResource() {
+        return "SSHCasCConfig.yml";
```

Comment:
FTR: If you rename the yml file to _configuration-as-code.yaml_ you don't need to override that.

Reply:
I have two files. I prefer each on to have a explicit name.